diagnostic.md

```diff
@@ -177,5 +202,6 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+At first I thought it would be 25px wide.
+However, now that I look at it more I think it won't be measurable in px at all. It would override the original px value and would be 25% of the page. It would only be 25px if it was in a parent div or class that was 100px.
```

Comment:
137.5 px - Reason being that since it's min-width, the media query will have .stretch be 25% and at 550px width, 25% is 137.5 px.